api/proto/info/info.proto

```diff
@@ -258,6 +264,7 @@ message ZInfoDevice {
   DataSecAtRest dataSecAtRestInfo = 29; //Info about Data At Rest Security
 
   ZInfoConfigItemStatus configItemStatus = 31;
+  repeated ZInfoAppInstance appInstances = 41;
```

Comment:
Probably better to pack the tags i.e. pick 32.

Reply:
I would like to put related fields together as much as possible, so leave a bit of gap between unrelated fields for additions .. Hence did this.

Reply:
Changed it to 32.